daktari/checks/onepassword.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import re
+from typing import Optional
+import json
+
+from daktari.check import Check, CheckResult
+from daktari.command_utils import get_stdout
+from daktari.os import OS
+from pathlib import Path
+
+
+class OnePassInstalled(Check):
+    name = "onepass.installed"
+
+    def __init__(self, minimum_version: Optional[float] = None):
+        self.minimum_version = minimum_version
+        self.suggestions = {
+            OS.GENERIC: "Install OP (1Password CLI): "
+                        "https://support.1password.com/command-line-getting-started/#set-up-the-command-line-tool",
+        }
+
+    def check(self) -> CheckResult:
+        installed_version = get_op_version()
+        return self.validate_minimum_version("op", installed_version, self.minimum_version)
+
+
+version_pattern = re.compile("([0-9]+.[0-9]+)")
+
+
+def get_op_version() -> Optional[float]:
+    raw_version = get_stdout("op --version")
+    if raw_version:
+        match = version_pattern.search(raw_version)
+        if match:
+            version_string = match.group(1)
+            logging.debug(f"OP Version: {version_string}")
+            return float(version_string)
+    return None
+
+
+class OPAccountExists(Check):
+    depends_on = [OnePassInstalled]
+
+    def __init__(self, context_name: str):
+        self.context_name = context_name
+        self.name = f"op.accountExists.{context_name}"
+        self.suggestions = {
+            OS.GENERIC: f"<cmd>op signin {context_name}.onepassword.com <your-email-here></cmd>",
+        }
+
+    def check(self) -> CheckResult:
+
+        home = str(Path.home())
+        if OS.OS_X:
```

Comment:
This is always true as written :sweat_smile: 

```suggestion
        if detect_os() == OS.OS_X:
```

Reply:
Oh flip sake haha.